cbr.tf

```diff
@@ -0,0 +1,15 @@
+##############################################################################
+# Update existing CBR VPC network zone
+##############################################################################
+
+resource "ibm_cbr_zone_addresses" "add_slz_vpc_crns" {
```

Comment:
why are we not using our module?

Reply:
[PR](https://github.com/terraform-ibm-modules/terraform-ibm-cbr/pull/530) raised and under review.

Reply:
Added CBR zone module.